draft-packages/collapsible/KaizenDraft/Collapsible/Collapsible.tsx

```diff
@@ -23,7 +25,7 @@ export type Props = {
   onToggle?: (open: boolean, id: string) => void
 
   /* Highlights the header when open by changing the background color */
```

Comment:
Can you update this comment to better describe what the new prop does?

Reply:
@talbet Done!